Code/Framework/AzCore/AzCore/Script/ScriptSystemComponent.cpp

```diff
@@ -404,6 +410,11 @@ ScriptLoadResult ScriptSystemComponent::LoadAndGetNativeContext(const Data::Asse
     }
     info.m_tableReference = ref;
 
+    SCRIPT_SYSTEM_SCRIPT_STATUS("ScriptSystemComponent"
+        , "ScriptSystemComponent::LoadAndGetNativeContext loaded from asset script: %s-%s"
+        , asset.GetHint().c_str()
+        , asset.GetId().ToString<AZStd::string>().c_str());
```

Comment:
Repeat: AZStd::fixed_string can be used here
```suggestion
        , asset.GetId().ToString<AZStd::fixed_string<AZ::Uuid::MaxStringBuffer>>().c_str());
```

Reply:
Done